packages/react-conformance/src/conformance.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
```

Comment:
Probably call this file `isConformant.tsx` (tsx since it will eventually contain jsx probably)